content/1_General/Debugging_General.mdx

```diff
@@ -29,6 +29,7 @@ todo - find particularly bad example of repetition
 </IncompleteSection>
 
  - Don't repeat yourself!
+   - Repetition usually results in a narrow mindset that slows the generation new/correct solutions.
```

Comment:
is this sentence necessary? :O

Reply:
Not necessary. But I thought the todo wanted examples.